ui-react/packages/atlasmap/src/UI/ConditionalExpressionInput.tsx

```diff
@@ -1,30 +1,106 @@
 import React, { FunctionComponent } from "react";
-import { ToolbarGroup, ToolbarItem, Button } from "@patternfly/react-core";
-import { css, StyleSheet } from "@patternfly/react-styles";
 import {
-  // ExpressionContent,
-  IExpressionContentProps,
-} from "./ExpressionContent";
+  ToolbarGroup,
+  ToolbarItem,
+  Button,
+  Tooltip,
+} from "@patternfly/react-core";
+import { css, StyleSheet } from "@patternfly/react-styles";
+import { ExpressionContent } from "./ExpressionContent";
 
 const styles = StyleSheet.create({
   toolbarItem: { flex: 1 },
 });
+export interface IConditionalExpressionInputProps {
```

Comment:
This interface should extend `IExpressionContentProps` to avoid re-declaring the same stuff twice.

Reply:
will address in a new issue